src/Authentication/OAuth20.php

```diff
@@ -130,10 +132,7 @@ public function authenticateClient(RestClient $client)
         }
     }
 
-    /**
-     * @param array|object $definitions
-     */
-    protected function addGenerator(AbstractSignature $subscriber, $definitions, array $authorization)
+    protected function addGenerator(AbstractSignature $subscriber, array $definitions, array $authorization): void
```

Comment:
`$definitions` už nemůže být object?

Reply:
Vdaka, ze si si to vsimol, zasluzi si to komentar.
Nemoze, podrobnou analyzou som zistil, ze to bude vzdy pole.

- je to protected metoda, ktora sa pouziva iba v tejto triede.
- dostane sa do nej vysledne bud `$this->query` alebo `$this->headers` -> obidva su polia + `arrayToObject` ocakava pole.

Vid kluc `definitions` tu:
https://github.com/keboola/generic-extractor/blob/d1487a5316300c1623134175d04341c0f825d9fc/src/Authentication/OAuth20.php#L99-L108

Reply:
super 👍 díky